recipes/uritemplate.py/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "uritemplate.py" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 4dd14904ba502c6ff8d6276e004404de
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Uses `distutils` in [this release]( https://github.com/sigmavirus24/uritemplate/blob/0.3.0/setup.py#L10 ). However, in `master` this will switch to using `setuptools`. So, we should keep an eye on that.